src/app/me/page.tsx

```diff
@@ -227,6 +234,30 @@ const Content = () => {
           </FormControl>
         </Box>
       </Flex>
+      <Flex gap="1rem">
+        <Box>
+          <FormControl as={SimpleGrid} columns={{ base: 2, lg: 2 }}>
+            <FormLabel htmlFor='advanceConfig' style={formLabelStyles}>
+              Are you <Link href="https://vectara.com/pricing/" isExternal textDecoration="underline">
```

Comment:
I thought we were going to just pull this from the Vectara API instead of surfacing it as a question?

Reply:
> I thought we were going to just pull this from the Vectara API instead of surfacing it as a question?

I think lets keep it simple for now. we can update it in the upcoming version.